### ISSUE TITLE
Cannot both subscribe and publish in the same process

### ISSUE THREAD

**Issue by Unitech** [dashersw/cote#11]
I know it can sounds weird (it's about multiple processes and microservice I know! :) ), but I'm trying to subscribe and publish in the same process but that does not work with this example:

```javascript
var Subscriber = require('cote').Subscriber;

var randomSubscriber = new Subscriber({
  name: 'randomSub',
  // namespace: 'rnd',                                                                                                                                                                                                                                                  
  subscribesTo: ['randomUpdate']
});

randomSubscriber.on('randomUpdate', function(req) {
  console.log('notified of ', req);
});

var Publisher = require('cote').Publisher;

// Instantiate a new Publisher component.                                                                                                                                                                                                                               
var randomPublisher = new Publisher({
  name: 'randomPub',
  // namespace: 'rnd',                                                                                                                                                                                                                                                  
  broadcasts: ['randomUpdate']
});

// Wait for the publisher to find an open port and listen on it.                                                                                                                                                                                                        
randomPublisher.on('ready', function() {
  setInterval(function() {
    var val = {
      val: ~~(Math.random() * 1000)
    };

    console.log('emitting', val);

    // publish an event with arbitrary data at any time                                                                                                                                                                                                                 
    randomPublisher.publish('randomUpdate', val);
  }, 3000);
});
```

Is there something I need to change?

**Comment by dashersw** [comment]
Hello @Unitech, this behavior was intended to prevent intra-process daemons from overriding the same UDP port, which, I presume, was a mis-behavior on Node's part. It seems to be removed, and node-discover has updated itself to allow intra-process daemons. So I published cote 0.6.1 to npm, which resolves this issue.

Your example now works. Cheers!

**Comment by Unitech** [comment]
It works! Thanks!!
